### PR TITLE
add release ci

### PR DESCRIPTION
**Pull request by jianyun8023**
Change the release repo to maven central repo.
Add release ci. (Release-ci will be disabled)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -267,22 +268,84 @@
...
+          </plugin>
+          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <version>1.6.8</version>
+            <extensions>true</extensions>
+            <configuration>
+              <serverId>ossrh</serverId>
+              <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+              <autoReleaseAfterClose>false</autoReleaseAfterClose>
```
Comment (jianyun8023) [inline pom.xml:344]: @codelipenghui The configuration for automatic close is here.

File: managed-ledger-shaded/pom.xml [hunk 2]
```diff
@@ -298,13 +298,55 @@
...
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-jar-plugin</artifactId>
+            <version>3.1.0</version>
```
Comment (syhily) [inline managed-ledger-shaded/pom.xml:317]: The plugin version should be moved to root pom.xml.

File: managed-ledger-shaded/pom.xml [hunk 3]
```diff
@@ -298,13 +298,55 @@
...
+                <id>javadoc-jar</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>jar</goal>
+                </goals>
+                <configuration>
+                  <classifier>javadoc</classifier>
+                </configuration>
+              </execution>
+              <!--              <execution>-->
```
Comment (syhily) [inline managed-ledger-shaded/pom.xml:336]: The commented code could just be removed.

File: pom.xml [hunk 4]
```diff
@@ -14,7 +14,8 @@
     limitations under the License.
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (syhily) [inline pom.xml:18]: This line wrap could be applied to all the pom.xml files for same code style. 🤣

File: pom.xml [hunk 5]
```diff
@@ -267,22 +268,84 @@
...
-    <repository>
-      <id>central</id>
-      <layout>default</layout>
-      <url>https://repo1.maven.org/maven2</url>
-    </repository>
-    <repository>
-      <id>bintray-streamnative-maven</id>
-      <name>bintray</name>
-      <url>https://dl.bintray.com/streamnative/maven</url>
-    </repository>
```
Comment (syhily) [inline pom.xml:280]: Dropping this repository means we can't rely on our streamnative's pulsar as before. Is this accepted?

File: pom.xml [hunk 6]
```diff
@@ -267,22 +268,84 @@
...
   </distributionManagement>
+  <profiles>
+    <profile>
+      <id>release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-source-plugin</artifactId>
+            <version>3.0.1</version>
```
Comment (syhily) [inline pom.xml:289]: Move all the version to properties field.

File: pulsar-flink-connector-shade-2.11/pom.xml [hunk 7]
```diff
@@ -152,5 +153,37 @@
...
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-jar-plugin</artifactId>
+            <version>3.1.0</version>
```
Comment (syhily) [inline pulsar-flink-connector-shade-2.11/pom.xml:163]: The plugin version should be moved to root pom.xml.

File: pulsar-flink-connector-shade-2.12/pom.xml [hunk 8]
```diff
@@ -132,5 +133,37 @@
...
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-jar-plugin</artifactId>
+            <version>3.1.0</version>
```
Comment (syhily) [inline pulsar-flink-connector-shade-2.12/pom.xml:143]: The plugin version should be moved to root pom.xml.

**Review by syhily — CHANGES_REQUESTED** [review]

File: managed-ledger-shaded/pom.xml (hunk 2 above)
Comment (jianyun8023) [reply managed-ledger-shaded/pom.xml:317]: Here is the module to make shade, can generate an empty javadoc package

File: managed-ledger-shaded/pom.xml (hunk 3 above)
Comment (jianyun8023) [reply managed-ledger-shaded/pom.xml:336]: ok.

File: pom.xml (hunk 5 above)
Comment (jianyun8023) [reply pom.xml:280]: Our future releases will be released in the maven central warehouse, and there is no need to rely on the bintray warehouse.

File: pom.xml (hunk 4 above)
Comment (jianyun8023) [reply pom.xml]: ok

**Review by syhily — APPROVED** [review]

**Pull request merged**